test/utils/utils.cpp

```diff
@@ -57,3 +57,48 @@ TEST_F(test, utils_parse_var) {
     EXPECT_TRUE(utils_parse_var("test1;test2;test3,abc;test4", "test3", &arg));
     EXPECT_TRUE(utils_parse_var("test1;test2;test3;test4,abc", "test4", &arg));
 }
+
+TEST_F(test, utils_translate_mem_visibility_flag) {
+    umf_memory_visibility_t in_flag = static_cast<umf_memory_visibility_t>(0);
+    unsigned out_flag;
+    auto ret = utils_translate_mem_visibility_flag(in_flag, &out_flag);
+    EXPECT_EQ(ret, UMF_RESULT_ERROR_INVALID_ARGUMENT);
+}
+
+TEST_F(test, utils_shm_open) {
+    auto ret = utils_shm_open(NULL);
+    ASSERT_EQ(ret, -1);
+
+    ret = utils_shm_open("invalid_path");
+    ASSERT_EQ(ret, -1);
+}
+
+TEST_F(test, utils_get_file_size) {
+    size_t size;
+    auto ret = utils_get_file_size(-1, &size);
+    ASSERT_EQ(ret, -1);
+
+    int fd = utils_create_anonymous_fd();
+    ret = utils_get_file_size(fd, &size);
+    ASSERT_EQ(ret, 0);
+    ASSERT_EQ(size, 0);
+}
+
+TEST_F(test, utils_set_file_size) {
```

Comment:
usually we add in the name information that the test is negatve, e.g. here: `utils_set_wrong_file_size` - pls apply to all new test cases